### PR TITLE
Add color offset

### PR DESCRIPTION
**Pull request by robert-crandall**
- Added option for sunrise/sunset offsets to be different for brightness and color

### CODE REVIEW

File: hacs.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
-    "name": "adaptive_lighting",
+    "name": "adaptive_lighting_color_offset",
```
Comment (basnijholt) [inline hacs.json:2]: Is this left over from trying it?

File: README.md [hunk 2]
```diff
@@ -1,194 +1,7 @@
 # Adaptive Lighting component for Home Assistant
 
-![](https://github.com/home-assistant/brands/raw/b4a168b9af282ef916e120d31091ecd5e3c35e66/core_integrations/adaptive_lighting/icon.png)
```
Comment (basnijholt) [inline README.md:3]: Deleted README text by accident?

**Comment by basnijholt** [comment]
This looks great! Will have a more detailed look later.

**Comment by robert-crandall** [comment]
Sorry @basnijholt! I opened this PR in error. I meant to merge it to my own branch 🤦

**Pull request closed without merging**